src/bats-mock.bash

```diff
@@ -57,7 +69,45 @@ fi
 EOF
   chmod +x "${mock}"
 
-  echo "${mock}"
+  if [[ -n "${cmd}" ]]; then
+    echo "${cmd}"
+  else
+    echo "${mock}"
+  fi
+}
+
+# Performs cleanup of mock objects
+# Globals:
+#   BATS_TMPDIR
+mock_teardown() {
+    rm -rf "${BATS_TMPDIR}"/bats-mock.$$.*
```

Comment:
I could not find a unit test for `mock_teardown`. I think it would be really easy to provide one